doc/book/src/introduction.md

```diff
@@ -30,6 +30,8 @@ to implement containers.
 
 ## Structure of This Book
 
+I think even before you get to "common concepts" you should have a "run your first Maelstrom test" section (something like [this part](https://docs.pytest.org/en/7.1.x/getting-started.html#create-your-first-test) of the Pytest documentation. Show me how easy it is to improve my sad, poorly performing test suite by using `cargo-maelstrom` to parallelize my tests locally. Then, perhaps, step back a bit and fill me in about what's going on under the covers with the "common concepts" section followed by a more in-depth treatment of `cargo-maelstrom`. Then blow me away by telling me how I can share this awesomeness with my whole team by deploying a cluster.
```

Comment:
This is general feedback...

Reply:
I've been wondering about this. I've thought it would be nice to have completely separate "getting started" book that builds up from a small example, and then hits on most of the highlights.

@bobbobbio had the idea of using an existing repo (somebody else's), instead of a toy example one.

I think that would fulfill this need, but maybe we should also have a slimmed down chapter like you suggest even if we have the larger guide?